Apply Kubernetes style guide improvements while preserving Bluefin's quirky personality (#235)
ublue-os/bluefin-docs: docs/command-line.md

### Message of the Day and `fastfetch`

The project prefers to have functional bling that is slick but it must also serve a purpose. New terminals (<kbd>Ctrl</kbd>+<kbd>Alt</kbd>+<kbd>Enter</kbd>) display a message of the day with some basic information:

![image](https://github.com/user-attachments/assets/0e0326ef-6640-41a2-bd24-dae1b1647cfd)

The `bluefin-dx:beta` line is the name of the OS image, and is a good way to remind yourself if you're on a pinned image as well as a quick reference to common commands. You can toggle it on and off with `ujust toggle-user-motd`. Note that the Tip rotates regularly in order to spread useful tips to the community.

> The project loves to flex machines and software. Run `fastfetch`:

Reviewer: ```suggestion
We love to flex our machines and software. Run `fastfetch`:
```
